pkg/kubernetes/scripts/date.js

```diff
@@ -24,13 +24,31 @@
 
     angular.module('kubernetes.date', [])
 
+    .directive('refreshEveryMin', [
+        '$interval',
+        function($interval) {
+            return {
+                restrict: 'E',
+                link: function(scope, element, attrs) {
+                    var interval = $interval(function () {}, 60000);
```

Comment:
Well having the loader trigger callbacks, has the same effect as just calling rootScope.digest() every once in a while. No? In which case I think we should just do the later rather than bringing the loader into this.